src/app/features/notifications/model/notifications-activity.ts

```diff
@@ -0,0 +1,18 @@
+import { Entity } from "../../../core/entity/model/entity";
+import { DatabaseField } from "../../../core/entity/database-field.decorator";
+import { DatabaseEntity } from "../../../core/entity/database-entity.decorator";
+
+@DatabaseEntity("Notification")
+export class NotificationActivity extends Entity {
```

Comment:
```suggestion
/**
 * This represents one specific notification event for one specific user,
 * displayed in the UI through the notification indicator in the toolbar.
 */
@DatabaseEntity("NotificationEvent")
export class NotificationEvent extends Entity {
```

Let's discuss the most clear naming. I am thinking:
- NotificationEvent = single, specific notification entry
- NotificationRule (or maybe NotificationSubscription or ....) = one configuration of a certain notification type for which the user wants to see events
- ... maybe there are other entities or interfaces also in the overall picture but I can't think of any right now.

What do you think? Please suggest alternative namings also.